### ISSUE TITLE
`os.DirFS()` problem on Windows

### ISSUE THREAD

**Issue by gysddn** [progrium/env86#7]
In many places, to check if a file exists, this call will be made:

```
fs.Exists(os.DirFS("/"), strings.TrimPrefix(pathToCheck, "/"))
```

The behavior here seems to be (needs checking) that `go` will open the *current* drive as the root, if the file is located in another drive this check will always fail.

A possible fix is:

- Check `GOOS` to see if on Windows
- Get the absolute path
- Get the drive name from the path
- Open the root as that drive

**Comment by progrium** [comment]
Yes, if on Windows, try to get the drive from the path to check.
